lib/android/app/src/main/java/com/reactnativenavigation/react/NavigationModule.java

```diff
@@ -72,6 +73,11 @@ public String getName() {
         return NAME;
     }
 
+    @ReactMethod
+    public void getLaunchArgs(String commandId,Promise promise) {
```

Comment:
nit pick: there is a space missing between `,` and `Promise`

Reply:
Not anymore 🙈